src/SpectatorView.Unity/Assets/SpectatorView/Prefabs/Default Mobile UI.prefab

```diff
@@ -1782,6 +1784,136 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   spatialAlignmentStateText: {fileID: 6067051665112448766}
+--- !u!1 &6750118295539874624
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 4674793626925419294}
+  - component: {fileID: 3447970668130146125}
+  m_Layer: 0
+  m_Name: Asset State Visual
+  m_TagString: Untagged
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
+--- !u!224 &4674793626925419294
+RectTransform:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 6750118295539874624}
+  m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
+  m_LocalPosition: {x: 0, y: 0, z: 0}
+  m_LocalScale: {x: 1, y: 1, z: 1}
+  m_Children: []
+  m_Father: {fileID: 3896832437504863526}
+  m_RootOrder: 0
+  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: 0, y: 0}
+  m_SizeDelta: {x: 100, y: 100}
+  m_Pivot: {x: 0.5, y: 0.5}
+--- !u!114 &3447970668130146125
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 6750118295539874624}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 8f305a7aab142dd4f86567c87318f58c, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  assetStateText: {fileID: 577653725205976457}
+--- !u!1 &7939958663571977771
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 2975881544335965903}
+  - component: {fileID: 731525659962201783}
+  - component: {fileID: 577653725205976457}
+  m_Layer: 0
+  m_Name: Asset State Text
```

Comment:
I added an extra "Asset State Text" text object above the other two state lines and moved them down to make room (this one can span to 2 lines!).